docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,432 @@
+.. currentmodule:: discord
+
+
+Errors and Error Handling
+==========================
+
+Handling Errors
+---------------
+
+When an error ocurrs in am ext.commands command, a client event, or a bot listener, it will by default show an error in the console 
```

Comment:
```suggestion
When an error occurs in an ext.commands command, a client event, or a bot listener, it will by default show an error in the console 
```